README.md

```diff
@@ -593,6 +615,19 @@ that refers to them.
 ### Why are some typedefs not generated?
 
 The following typedefs are not generated -
-- They are not referred to anywhere in the included declarations.
-- They refer to a struct/union having the same name as itself.
-- They refer to a boolean, enum, inline array, Handle or any unsupported type.
+  - They are not referred to anywhere in the included declarations.
+  - They refer to a struct/union having the same name as itself.
+  - They refer to a boolean, enum, inline array, Handle or any unsupported type.
```

Comment:
Why indent these, but not indent the ones under the next header?

Reply:
Fixed.